app/mailers/candidate_mailer.rb

```diff
@@ -78,7 +78,10 @@ def application_rejected_all_rejected(application_choice)
 
     email_for_candidate(
       application_choice.application_form,
-      subject: I18n.t!('candidate_mailer.application_rejected.all_rejected.subject', provider_name: @course.provider.name),
+      subject: I18n.t!(
+        "candidate_mailer.application_rejected#{application_choice.rejected_by_default ? '_by_default' : ''}.all_rejected.subject",
```

Comment:
Since they're all the same, perhaps this is clearer:

```suggestion
        application_choice.rejected_by_default ? "candidate_mailer.application_rejected_by_default.subject" : "candidate_mailer.application_rejected.all_rejected.subject",
```